data/assets/examples/renderable/renderablenodearrow/renderablenodearrow_relativevalues.asset

```diff
@@ -0,0 +1,40 @@
+-- This example shows an arrow pointing from one scene graph node in the direction of
+-- another, but where the size is specified using relative values (based on the bounding
+-- sphere of the start node). Here it points from Earth to the Moon
+--
+-- Note that the arrows are generated as objects in 3D space  and need to have a size
+-- that is suitable for their 3D context. Here it is set based on the size of the Earth.
+
+local earth = asset.require("scene/solarsystem/planets/earth/earth")
+local moon = asset.require("scene/solarsystem/planets/earth/moon/moon")
+
+local EarthRadius = 6371000.0
+
+local Node = {
+  Identifier = "RenderableNodeArrow_Example_Relative",
+  Parent = earth.Earth.Identifier,
+  Renderable = {
+    Type = "RenderableNodeArrow",
+    StartNode = earth.Earth.Identifier,
+    EndNode = moon.Moon.Identifier,
+    UseRelativeOffset = true,
+    UseRelativeLength = true,
+    -- Specify relative values (times the size of Earth, in this case)
+    Offset = 2.0,
+    Length = 5.0,
+    -- Width is still in meters
```

Comment:
```suggestion
    -- Width is in meters
```

Reply:
"still" was added here to point out that while the offset and length are in relative units, the width is still specified in meters